package/kernel/linux/modules/crypto.mk

```diff
@@ -992,6 +994,17 @@ define KernelPackage/crypto-sha256
   $(call AddDepends/crypto)
 endef
 
+define KernelPackage/crypto-sha256/aarch64
+  FILES+=$(LINUX_DIR)/arch/arm64/crypto/sha256-arm64.ko
+  AUTOLOAD+=$(call AutoLoad,09,sha256-arm64)
+endef
+
+define KernelPackage/crypto-sha256/aarch64-ce
```

Comment:
Shouldnt be this as ``crypto-sha2/aarch64-ce``? Same as the file at 1004?

Basically, recently you added sha1 and it seems that it does not match with this one , which you packaged recently as it is different for sha2-ce. https://github.com/openwrt/openwrt/commit/9425064bdbc873a285f476da101199433d3a80e4

Reply:
Ok, ok! Thanks for clarification. I was just asking about it, because it seemed weird to me.